packages/input/src/calcTextareaHeight.js

```diff
@@ -70,7 +70,9 @@ export default function calcTextareaHeight(
   hiddenTextarea.setAttribute('style', `${contextStyle};${HIDDEN_STYLE}`);
   hiddenTextarea.value = targetElement.value || targetElement.placeholder || '';
 
-  let height = hiddenTextarea.scrollHeight;
+  // Fixes scrollbar appearing when resizing
+  // scroll height returns an int, this lack of precision might be the cause, while testing it looked to be correlated to the pixel ratio
+  let height = hiddenTextarea.scrollHeight + (window.devicePixelRatio < 1 ? 1 : window.devicePixelRatio - 1);
```

Comment:
Shouldn't it be `window.devicePixelRatio < 1 ? 0 : ...`

Reply:
No because the scrollbar was still showing so it needs a little bit extra height, I believe I saw it on my 1080p monitor at 75% so devicePixelRatio is .75 if you want to try it out.